packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -310,6 +312,16 @@ export type VideoSource =
        * @platform ios
        */
       useCaching?: boolean;
+
+      /**
+       * Specifies the content type of the video source. When set to 'auto' the player will try to automatically determine the content type.
```

Comment:
```suggestion
       * Specifies the content type of the video source. When set to `'auto'`, the player will try to automatically determine the content type.
```

Missing comma and inline syntax highlight.